checklists/alz_checklist.en.json

```diff
@@ -1630,17 +1586,6 @@
             "severity": "High",
             "link": "https://learn.microsoft.com/azure/governance/policy/overview"
         },
-        {
-            "category": "Governance",
-            "subcategory": "Governance",
```

Comment:
Is the idea for this one that C02.14 covers this material?  Could we include the append policy mode to that one?

Or is the recommendation now not to use append policy for required tags?

Reply:
What I'm thinking is I leave this one here but remove C02.14, what are your thoughts?

Reply:
@BeckyHope19 and @aarunraaj  - What are your thoughts?

I can see one check for "Identify required tags, including billing and cost management, and use the 'append' policy..." could be a single recommendation, but I don't know if there is a reason to track them separately, I am not thinking of.